### ISSUE TITLE
SBI Watchdog extension?

### ISSUE THREAD

**Issue by andreiw** [riscv-non-isa/riscv-sbi-doc#152]
I asked in https://github.com/riscv-non-isa/server-soc/issues/47 about a hardware watchdog, as feature parity-wise this is a gap, when compared to AArch64 servers. Ved pointed me at the SBI TG. Has any work started on a watchdog extension?

**Comment by atishp04** [comment]
@andreiw : No. There was no discussion about SBI watchdog extension. Is there a hardware spec for watchdog that needed an SBI extension ? I found this but I don't think it was ratified or agreed upon.

https://github.com/riscvarchive/riscv-watchdog/blob/main/riscv-watchdog.adoc

@avpatel for confirmation.
